reporter/src/main/kotlin/reporters/AbstractNoticeReporter.kt

```diff
@@ -87,6 +96,36 @@ abstract class AbstractNoticeReporter : Reporter() {
         return result.toSortedMap()
     }
 
+    private fun postProcess(ortResult: OrtResult, postProcessingScript: String, noticeReport: NoticeReport)
```

Comment:
Nit: Maybe make `noticeReport` the first argument, so that the logic is: Input first, additional input second, then the algorithm, returning the output.